src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const tempState = { ...state };
+  const statesArr = [];
```

Comment:
```suggestion
  const statesHistory = [];
```
`arr` in name doesn't make much sense